test/shell/test_coverage_scalatest_resources.sh

```diff
@@ -5,7 +5,9 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 test_coverage_succeeds_resource_call() {
-    bazel coverage //test/coverage_scalatest_resources/consumer:tests
+    bazel coverage \
```

Comment:
@fmeum thanks for contribution! Why you had to change this test? Could add new one with `--instrumentation_filter` and leave this one as is. As I understand coverage should work without `--instrumentation_filter` as well.

Reply:
It generally does, but not in this situation: Bazel auto-detects the `--instrumentation_filter` as `^//test/coverage_scalatest_resources/consumer[:/]`, which is not sufficient to pass this test as instrumenting `//test/coverage_scalatest_resources/resource` is required to do so. 

This could also happen to users of this ruleset and is thus a mild version of a breaking change, but then `--instrumentation_filter` is only meant to be a best-effort guess in the first place. I can add information about this to the README or some other place in the docs if you prefer.

Reply:
Ah wasn't aware of bazel's `auto-detect`. Sorry for the noise.